### PR TITLE
Create a popup instead of category create update and delete pages

### PR DESCRIPTION
**Pull request by Otabek7667a**
I create delete  and create and update modal for category
image
image
image
image

### CODE REVIEW

File: ExpenseTracker/ExpenseTracker.Application/ViewModels/Category/CategoryViewModel.cs [hunk 1]
```diff
@@ -6,8 +6,9 @@ public class CategoryViewModel
 {
     public int Id { get; init; }
     public required string Name { get; init; }
-    public string? Description { get; init; }
+    public string Description { get; init; }
```
Comment (jamshidbek7375) [inline ExpenseTracker/ExpenseTracker.Application/ViewModels/Category/CategoryViewModel.cs:9]: why did you do that

File: ExpenseTracker/ExpenseTracker/Views/Categories/Index.cshtml [hunk 2]
```diff
@@ -87,5 +91,76 @@
                 grid.refresh();
             }
         });
-    }   
-</script>
\ No newline at end of file
+    }
+    
+</script>
+
```
Comment (jamshidbek7375) [inline ExpenseTracker/ExpenseTracker/Views/Categories/Index.cshtml:97]: one line is not needed

File: ExpenseTracker/ExpenseTracker/Views/Categories/Index.cshtml [hunk 3]
```diff
@@ -87,5 +91,76 @@
...
+        }
+    }
+</script>
+
+
+
+
+
+
+
```
Comment (jamshidbek7375) [inline ExpenseTracker/ExpenseTracker/Views/Categories/Index.cshtml:166]: one lines is not needed

File: ExpenseTracker/ExpenseTracker/appsettings.json [hunk 4]
```diff
@@ -5,5 +5,17 @@
...
+  "MailSettings": {
+    "From": "otabekxudoynazarov131@gmail.com",
+    "SmtpServer": "smtp.gmail.com",
+    "Port": 465,
+    "Username": "otabekxudoynazarov131@gmail.com",
+    "Password": "sixq trwl zebo xfza"
+  },
+
+  "SyncfusionKey": "Ngo9BigBOggjHTQxAR8/V1NCaF5cXmZCf1FpRmJGdld5fUVHYVZUTXxaS00DNHVRdkdnWXdedHRQRGRZWEV+WkQ="
 }
```
Comment (jamshidbek7375) [inline ExpenseTracker/ExpenseTracker/appsettings.json:21]: don't download this

**Review by jamshidbek7375 — CHANGES_REQUESTED** [review]

**Comment by Otabek7667a** [comment]
Ok I accidently did it

Shan, 19-okt, 2024, 13:00 Jamshidbek ***@***.***>:

**Pull request closed without merging**